src/examples/titlePanel/index.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html>
+<head>
+	<title>TitlePanel Example</title>
+	<link rel="stylesheet" href="../../styles/widgets.css">
+	<style>
+		body {
+			font-family: Calibri, Arial, sans-serif;
+		}
+
+		.main-app {
+			margin: 20px;
+			width: 800px;
+		}
+
+		.main-app > div {
+			margin-bottom: 15px;
+		}
+	</style>
+</head>
+
+<body>
+	<script src="../../../../node_modules/@dojo/loader/loader.min.js"></script>
+	<!-- TODO: should come from project dependency -->
+	<script src="https://cdnjs.cloudflare.com/ajax/libs/maquette/2.4.2/css-transitions.min.js"></script>
```

Comment:
I've seen this elsewhere in the codebase but i'll ask anyway, is there any reason this cannot be loaded from `node_modules/maquette/dist/css-transitions.min.js`?